### DISCUSSION TITLE
Can we use e.g. '/dashboard' instead of '/admin' ?

### DISCUSSION THREAD

**Discussion by willemmulder** [payloadcms/payload#10163] [Q&A] [answered]
I want to run payload from a custom folder / path instead of the default `/admin`. Is that possible?

If I try it locally and rename the `admin `folder to `dashboard`, the initial navigation works fine, but all links still point to `/admin/users` etc.

Any clues?

**Comment by jacobsfletch** [comment, marked as answer]
This is possible. After changing your folder structure you'll also need to ensure that `routes.admin` matches in your config. It defaults to `/admin`. More on that here: https://payloadcms.com/docs/admin/overview#customizing-routes

**Reply by willemmulder** [reply]
Perfect, solved! 🏆
